### ISSUE TITLE
copy_message does not copy the reply_markup from the original message

### ISSUE THREAD

**Issue by SSxQ** [eternnoir/pyTelegramBotAPI#2421]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using? 4.24.0

2. What OS are you using? Windows 11 21H2

3. What version of python are you using? 3.10.8

I have a chat where I added a bot and made it an admin. The chat contains messages that contain inline buttons. Using the copy_message method, I wanted to copy the message with the buttons through the bot, but the body of the message is copied except reply_markup. Is this a library or telegram limitation? I did not find any restrictions in the documentation

**Comment by SSxQ** [comment]
At the same time, forward_message works fine, forwards messages along with buttons

**Comment by Badiboy** [comment]
I'll take a look if it is possible to save markup  on copy.

PS. Forward works on different principles.

**Comment by coder2020official** [comment]
No. This is a telegram limitation it seems. If you want to copy the reply markup too, you should pass it on as a parameter to copy_message.
